board/common/post-image.sh

```diff
@@ -42,7 +42,7 @@ fi
 load_cfg DISK_IMAGE
 if [ "$DISK_IMAGE" = "y" ]; then
     ixmsg "Creating Disk Image"
-    diskimg="${NAME}-disk.img"
+    diskimg="${NAME}-disk$(ver).img"
```

Comment:
No, nothing.  Should we go with symlinks instead, or did you have something else in mind?  Otherwise I think that should fall under the responsibility of the developer.